ui.R

```diff
@@ -16,7 +16,7 @@ shinyUI(
       # White header with logo
       tags$div(class = "header-white",
 
-               tags$img(src = "images/test1.png", style = "height: 101px; width: 242px; margin-bottom: 10px;margin-top: 10px; margin-right: 10px")
+               tags$img(src = "images/Banner_2018.png", style = "height: 120px; width: 610px; margin-bottom: 10px;margin-top: 10px; margin-right: 10px")
```

Comment:
Banner_2 is a bit wider, compared to the uploaded file in images/. Not a problem, just a note.